app/views/keywords/index.html.slim

```diff
@@ -1,10 +1,26 @@
-p[style="color: green"]
-  = notice
+section.keywords
+  .container
+    .keywords__top
+      img src="#{image_url 'nimblehq_logo.svg'}" alt="nimblehq"
+      .keywords__hero
+        h1 G-Scraper
+        p Scrape data right off Google!
 
-h1 Keywords
+        .keywords__line
 
-= render 'upload-csv-form', keyword: @keyword
+    .keywords__middle
+      .keywords__user
+        h3 Welcome
+        span #{current_user.email}
 
-#keywords
-  - keyword_presenter.each do |keyword|
-    = render 'keyword', keyword: keyword
+      - if user_signed_in?
+        .keywords__logout
+          = button_to 'Logout', destroy_user_session_path, method: :delete,
+            class: 'button button--secondary header__logout'
```

Comment:
Here also be able to turn into a new component such as a greeting component.
And the we will have some classes for `h3`, `span` and so on
The same purpose like the previous comment

Reply:
👍🏾 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/4de259ac3b8d20a892a061fd18c97a8a63517e55